package.json

```diff
@@ -128,7 +128,7 @@
     "ovh-angular-timeline": "^1.5.2",
     "ovh-angular-ui-confirm-modal": "^1.0.2",
     "ovh-angular-user-pref": "^0.3.1",
-    "ovh-api-services": "^6.1.2",
+    "ovh-api-services": "git+https://github.com/ovh-ux/ovh-api-services.git#feat/template-modem",
```

Comment:
@Steffy29  A new release of [`ovh-api-services`](https://github.com/ovh-ux/ovh-api-services/compare/v6.12.0...v6.13.0) has been published.

You can now refer to the `v6.13.0`.